Jenkinsfile

```diff
@@ -138,7 +138,7 @@ void runCheckoutPim17(String pimVersion) {
     node('docker') {
         deleteDir()
         try {
-            docker.image("akeneo/php:5.6").inside("-v /home/akeneo/.composer:/home/docker/.composer") {
+            docker.image("akeneo/php:5.6").inside("-u docker -v /home/akeneo/.composer:/home/docker/.composer") {
```

Comment:
not needed, except for integration tests

Reply:
I had pb in all pipeline so I put it everywhere!